content/news/march-2016-tournament-annoucment.md

```diff
@@ -0,0 +1,22 @@
+
+
+[March 2016 OpenRA Tournament](..\images\news\march-2016-tournament.jpg)
+
+If you are interested in some heated tag-team battles in OpenRA [Register Your Team Here] (http://www.sleipnirstuff.com/forum/viewtopic.php?f=82&t=19414)!  
+
+March 5th-6th and 11th - 12th 2016!
+
+Game Progression will be as follows: 
+Each player will first face off against an opponent in a 1v1 no-holds-barred fight to the death! 
+Then the teams will join forces in an attempt to wipe their adversary from the face of the planet in a 2v2 Superpower Scrap!!!    
+
+Thanks to the Comcast Crew and [GameReplays](http://www.gamereplays.org/portals.php) prize money is available for this tournament!  
+The team that comes in 1st place will receive $160.00 USD
+Second place will win $40.00 USD
+To be split evenly among the team members.
```

Comment:
These three points can probably be condensed into a sentence. If not, keep the reward amounts as a single sentence and add the team members point in another sentence.